docs/main/prerequisites.md

```diff
@@ -26,6 +30,13 @@ The following operating systems are **officially** supported:
 | Fedora       | 31 / 32          | ARM / x86_64        |
 | CentOS       | 7 / 8            | x86_64              |
 
+!!! info
+    Pi-hole will run on most Debian-based distributions and is the preferred platform.
```

Comment:
There isn't a preferred platform for Pi-hole really.

Reply:
alright, I got it from here in case you're wondering: https://discourse.pi-hole.net/t/hardware-software-requirements/273

Reply:
Thanks, that's a post from October of 2016, things have changed since then.

Reply:
Alright